Gemfile

```diff
@@ -32,7 +32,7 @@ gem 'attr_encrypted', '3.1.0'
 gem 'aws-sdk-s3', '~> 1'
 gem 'aws-sdk-sns', '~> 1'
 gem 'betamocks', git: 'https://github.com/department-of-veterans-affairs/betamocks', branch: 'master'
-gem 'breakers'
+gem 'breakers', git: 'https://github.com/department-of-veterans-affairs/breakers', branch: 'master'
```

Comment:
Are we not allowed to run a new release of the breakers gem?

Reply:
When running bundle update on breakers, I receive "Bundler attempted to update breakers but its version stayed the same" unless the repo/master branch is specified. Any ideas?

Reply:
I don't have maintainer permissions to build/push the new gem version but I emailed Aubrey Holland. 

https://rubygems.org/gems/breakers

Reply:
Aubrey was former Ad Hoc employee. I'm thinking this gem could probably be forked and made available in DSVA github.